include/zephyr/drivers/mfd/npm13xx.h

```diff
@@ -92,6 +92,20 @@ int mfd_npm13xx_reg_write(const struct device *dev, uint8_t base, uint8_t offset
 int mfd_npm13xx_reg_write2(const struct device *dev, uint8_t base, uint8_t offset, uint8_t data1,
 			   uint8_t data2);
 
+/**
+ * @brief Write multiple registers to npm13xx
+ *
+ * @param dev npm13xx mfd device
+ * @param base Register base address (bits 15..8 of 16-bit address)
+ * @param offset First register offset address (bits 7..0 of 16-bit address)
+ * @param len Number of bytes to write
+ * @param ... Bytes to write
+ * @retval 0 If successful
+ * @retval -errno In case of any bus error (see i2c_write_dt())
+ */
+int mfd_npm13xx_reg_write_burst(const struct device *dev, uint8_t base, uint8_t offset, size_t len,
```

Comment:
I'm a bit torn about the vararg here. The sonar quality check considers this a breach of MISRA, but it passes compliance anyway. Should it be changed in anticipation of future MISRA requirement?

Regardless, this generic function could replace the use of `mfd_npm13xx_reg_write2`?